tests/benchmark/data/serializer.py

```diff
@@ -133,15 +162,22 @@ def serialize(files=DEFAULT_FILES):
         # Finished indexing, going back an indentation level so we close the input file
 
         print('lastly, serializing the index')
-        output = os.path.join(location, serialized_file_name + '.hnsw_v3')
+        output = os.path.join(location, serialized_file_name + '.hnsw_v' + str(CUR_SERIALIZING_VERSION))
         index.save_index(output)
         print('saved to: ', output)
 
         # Sanity check - attempt to load the index
         index = HNSWIndex(output)
+        index.check_integrity()
 
         # Release memory before next iteration
         del index
 
 if __name__ == '__main__':
-    serialize()
+    serialize([{
```

Comment:
Did you leave it like this (glove-25 only) on purpose?

Reply:
nope. removing